twitter.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Twitter</title>
+    <script src="https://code.iconify.design/1/1.0.3/iconify.min.js"></script>
+    <link rel="stylesheet" href="master.css">
+</head>
+<body>
+    <section class="split left">
+        <img class="banner" src="./images/birdbackground.png">
+        <div>
+            <ul id="leftList">
+                <p><span class="iconify" data-icon="ps:magnifying-glass" data-inline="false"></span> Follow your interests.</p>
+                <p><span class="iconify" data-icon="ic:baseline-people-outline" data-inline="false"></span> Hear what people are talking about.</p>
+                <p><span class="iconify" data-icon="simple-line-icons:bubble" data-inline="false"></span> Join the conversation.</p>
+            </ul>
+        </div>
+    </section>
+    <section class="split right">
+        <div id="top">
+            <input type="text" placeholder="Phone, email, or username">
+            <input type="text" placeholder="Password">
+            <button id="submit" type="submit">Log In</button>
+        </div>
+        <div>
+            <p id="forgot">Forgot password?</p>
+        </div>
+        <div id="rightSide">
+            <div class="birdIcon">
+                <img class="birdImage" src="./images/birdlogo.png">
+            </div>
+            <h1>See what's happening in the world right now</h1>
+            <br>
+            <h3>Join Twitter today.</h3>
+            <button id="signUp">Sign Up</button>
+            <br>
+            <button id="logIn">Log In</button>
+        </div>
+
+    </section>
+</body>
+<footer class="footer">
```

Comment:
Since you're only using one footer, you can target `footer` when adding CSS, thus removing the need to add a class here.